templates/pipeline.jinja.html

```diff
@@ -773,12 +781,20 @@ <h1>
 
 
           {% if job["stdout"] %}
-          <div class="output-box"><xmp>stdout:
-            {% for line in job["stdout"] %}
-{{ line.strip() }}
-            {%- endfor %}{# line in job["stdout"] #}
-            </xmp>
-          </div><!-- class="output-box" -->
+            {% if job["wrapper_arguments"]["tags"] is not none and "literal-stdout" in job["wrapper_arguments"]["tags"] %}
+            <div class="literal-stdout">
+              {% for line in job["stdout"] %}
+  {{ line.strip() }}
```

Comment:
Can this be indented or would it affect the output?

Reply:
Indentations affects the output. This change will have to remain this way.